code/game/objects/structures/safe.dm

```diff
@@ -312,8 +312,9 @@ GLOBAL_LIST_EMPTY(safes)
 
 /obj/structure/safe/Destroy()
 	GLOB.safes -= src
-	drill.soundloop.stop()
-	return ..()
+	if(drill?.soundloop)

```

Comment:
It's preferable you use `if(drill && drill.soundloop)` instead of the new syntax operators.

Reply:
I'm happy to do so but can you explain why?

Reply:
Oh, it's just that it's rather weird syntax compared to other languages. Making use of it when it doesn't really accomplish anything but slightly shorter yet less readable if statements isn't great, IMO. If there was some ridiculous performance benefit or something to it, then sure, but otherwise, we might as well stick with what we've been stuck with.